src/Intercom/Clients/CompanyClient.cs

```diff
@@ -194,7 +194,10 @@ private String Transform(Company company)
                 name = company.name,
                 monthly_spend = company.monthly_spend,
                 custom_attributes = company.custom_attributes,
-                plan = company.plan != null ? company.plan.name : String.Empty
+                plan = company.plan != null ? company.plan.name : null,
```

Comment:
Does this change fix issue where we were over writing the company name with empty string when updating companies? Is that the benefit of having null here instead?

Reply:
That line of code is already in the repo, I just had to make that change there to avoid rebasing. Basically because we ignore null in our JsonSerializer, if the company.plan.name isn't set, we won't send it in the payload. 👍